api/resources/component.go

```diff
@@ -55,19 +58,53 @@ type Component struct {
 	Name                   string                 `json:"name"`
 }
 
+type CPUQuantity struct {
+	resource.Quantity
+}
+
+func (c *CPUQuantity) MarshalJSON() ([]byte, error) {
+	capInStr := strconv.FormatInt(c.MilliValue(), 10)
+	return []byte(fmt.Sprintf(`"%s"`, capInStr)), nil
+}
+
+type MemoryQuantity struct {
+	resource.Quantity
+}
+
+func (m *MemoryQuantity) MarshalJSON() ([]byte, error) {
+	capInStr := strconv.FormatInt(m.Value(), 10)
```

Comment:
Memory MarshaJSON is the same with CPUQuantity MarshalJSON. DRY

Reply:
- memory use: c.Value()
- cpu use: c.MilliValue()